packages/rspack/tests/configCases/entry/filename/test.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+	findBundle: function () {
```

Comment:
What is this file for?

Reply:
find test entry, default is main.js https://github.com/web-infra-dev/rspack/blob/17398d9fcea6d6b97b4ab65df46f066df231adea/packages/rspack/tests/ConfigCase.template.ts#L429